### DISCUSSION TITLE
Is it possible to flash it on any android device?

### DISCUSSION THREAD

**Discussion by OctavXD** [ItsRiprod/DeskThing#11] [Q&A]
I was thinking to flash it to an old android phone to get access to those features. Sounds like such a nice addition for the pc, but unfortunately I can't find any car things at a reasonable price in my country

**Comment by ItsRiprod** [comment]
Hello! I'm glad to see you're interested in this little project of mine.
Android? Unfortunately probably not. Mobile development is an absolute pain to get ported to. However, all hope isn't lost. On the CarThingHax discord server, we're working on collecting various parts (microcontroller, screen, buttons, etc) to build our own Car Thing that will have the same (if not more) functionality! Would definitely recommend checking that out.

Additionally, the DeskThing will eventually support a "widget" mode where it will show the Car Thing display but on your desktop (so while not its own physical device - it's more than nothing)

Now thinking about it - I may end up making this work on a local network (so an old Android phone may work-)

**Comment by OctavXD** [comment]
Glad to hear that! I was thinking about installing twrp (or another variant of it) on my old a32 5g and I was thinking there might be a compatibility between the your project files and the phone, making it boot your files instead of the android since I understood that the car thing is nothing but linux system (and I might be wrong, but I think android is also based on linux). I am kind of new to custom roms, flashing, rooting and stuff like that, and I really was hoping to start this adventure with your little project!
